src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/settings/ProblemSeverity.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+package com.redhat.devtools.intellij.lsp4mp4ij.settings;
+
+import com.intellij.lang.annotation.HighlightSeverity;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Problem severity levels used by LSP4MP
+ */
+//TODO move to lsp4ij?
+public enum ProblemSeverity {
```

Comment:
Why not using DiagnosticSeverity and null for none?

Reply:
DiagnosticSeverity has other values like hint and info, which could be mapped to IJ's severities, but since we don't use them, eventually I kept ProblemSeverity, as it makes things simpler, less risks of confusion.